pqlite-helm/templates/indexer/indexer.yaml

```diff
@@ -0,0 +1,126 @@
+{{- if .Values.shardCount }}
+{{- $replicas := .Values.shardCount | int }}
```

Comment:
here there seems to be a confusion between `replica` and `shard`

Reply:
yes, i will change the name